bin/browsertime.js

```diff
@@ -49,7 +50,23 @@ function run(url, options) {
       (scriptsByCategory, userScripts) => merge(scriptsByCategory, userScripts));
   }
 
-  engine.start()
+let xvfb;
+if (options.xvfb) {
+    const width = options.chrome ? options.chrome.mobileEmulation.width : undefined;
+    const height = options.chrome ? options.chrome.mobileEmulation.height: undefined;
```

Comment:
This will crash if --chrome.mobileEmulation.* isn't set. You could use lodash.get, and get(options, 'chrome.mobileEmulation.height')